File: python/fundamentals/day-08/Problem4.py
# File: python/fundamentals/day-08/problem4.py

numbers = [1, 2, 3, 5]

n = len(numbers) + 1  # Total count should be 5
expected_sum = n * (n + 1) // 2
actual_sum = sum(numbers)

missing = expected_sum - actual_sum
print(f"Missing: {missing}")

File: java/fundamentals/day08/Problem4.java
// File: java/fundamentals/day08/Problem4.java
package fundamentals.day08;

public class Problem4 {
    public static void main(String[] args) {
        int[] numbers = {1, 2, 3, 5};

        int n = numbers.length + 1;
        int expectedSum = n * (n + 1) / 2;
        int actualSum = 0;

        for (int num : numbers) {
            actualSum += num;
        }

        int missing = expectedSum - actualSum;
        System.out.println("Missing: " + missing);
    }
}